concourse/pipelines/windows-install-media-image-build.yaml

```diff
@@ -0,0 +1,202 @@
+---
+resource_types:
+- name: gcs
+  type: registry-image
+  source:
+    repository: frodenas/gcs-resource
+
+resources:
+- name: compute-image-tools
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/compute-image-tools.git
+    branch: master
+- name: guest-test-infra
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-test-infra.git
+    branch: master
+- name: windows-install-media-gcs
+  type: gcs
+  source:
+    bucket: artifact-releaser-prod-rtp
+    regexp: "windows-install-media/windows-install-media-v([0-9]+).tar.gz"
+
+jobs:
+# Build jobs
+- name: build-windows-install-media
+  plan:
+  - get: compute-image-tools
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: generate-build-id
+    file: guest-test-infra/concourse/tasks/generate-build-id.yaml
+    vars:
+      prefix: "windows-install-media"
+  - put: windows-install-media-gcs
+    params:
+      file: build-id-dir/windows-install-media*
+    get_params:
+      skip_download: "true"
+  - load_var: gcs-url
+    file: windows-install-media-gcs/url
+  - task: generate-build-date
+    file: guest-test-infra/concourse/tasks/generate-version.yaml
+  - load_var: build-date
+    file: publish-version/version
+  - task: daisy-build-windows-install-media
+    file: guest-test-infra/concourse/tasks/daisy-build-images-debian.yaml
+    vars:
+      wf: "windows_media/windows-install-media.wf.json"
```

Comment:
When the files and workflow are added to compute-image-tools, please place them in daisy_workflows/image_build/windows

The ISO and update file references  need to be changed to be required values passed in by concourse as all the values are in secret manager.

The publish file needs to be placed in daisy_workflows/build-publish)/windows and we should meet about updating the publish project that is currently being used.